packages/@uppy/companion/src/server/helpers/youtube_dl.js

```diff
@@ -17,7 +17,8 @@ function streamFile (url, isAudio, output) {
     noPlaylist: true,
     retries: 1,
 
-    proxy: 'https://sendheirloom_Gc208:NzAXHfgt_JAfh5c@dc.oxylabs.io:8000/',
+    cookies: process.cwd() + '/cookies.txt',
+    proxy: 'http://sendheirloom_Gc208:NzAXHfgt_JAfh5c@dc.oxylabs.io:8000/',
```

Comment:
the lack of https is unfortunate, but I already spent at least an hour on the Python 3.x upgrade and this was the quickest unblocker